### ISSUE TITLE
Python Function App YAML deployment Failed

### ISSUE THREAD

**Issue by MayankBargali-MSFT** [MicrosoftDocs/azure-devops-docs#12552]
Customer have reached out to Microsoft QnA as they deployed was failed: https://docs.microsoft.com/en-us/answers/questions/945144/index.html. As per the customer the commands are not correct that are documented.

Documented Build Command:
pip install -r requirements.txt

Customer updated below command to make it work
pip install --target="./.python_packages/lib/site-packages" -r ./requirements.txt

Customer has also specified final deployment task which was missing from document
deploymentMethod: zipDeploy 

Please review the thread and if there is document enhancement, please update the document.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 744eaf3c-85c6-2092-823a-8a709018b5f7
* Version Independent ID: 3ed6d4b4-334f-775b-f783-912b76124557
* Content: [Build and deploy Python web apps - Azure Pipelines](https://docs.microsoft.com/en-us/azure/devops/pipelines/ecosystems/python-webapp?view=azure-devops)
* Content Source: [docs/pipelines/ecosystems/python-webapp.md](https://github.com/MicrosoftDocs/azure-devops-docs/blob/main/docs/pipelines/ecosystems/python-webapp.md)
* Product: **devops**
* Technology: **devops-cicd-ecosystems**
* GitHub Login: @juliakm
* Microsoft Alias: **jukullam**

**Comment by juliakm** [comment]
Thanks @MayankBargali-MSFT! I just ran into the same issue and am updating the article.

**Comment by druskacik** [comment]
I ran into this issue when setting up continuous deployment of the Azure function from GitHub. I did not have a `.yml` file, so Azure auto-generated one for me. To fix the problem, I had to change the `pip install` command as specified above.

Maybe the auto-generated file could contain the correct command by default? However, I swear I used the same approach a few months ago and everything worked properly, so I don't know why the problem arose now.

![image](https://user-images.githubusercontent.com/47897205/211565273-a76dede5-dc2a-48cb-b8cd-84aa7740ad73.png)
